frontend/src/components/auth/auth.module.scss

```
@@ -4,11 +4,8 @@
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  background: (
-    url("/assets/img/decorate.svg") 0 0 no-repeat,
-    url("/assets/img/network.svg") 1685px 544px no-repeat,
-    #2a2a2a
-  );
+  background: (url("/assets/img/decorate.svg") 0 0 no-repeat, #2a2a2a);
```

```suggestion
  background: url("/assets/img/decorate.svg") 0 0 no-repeat, #2a2a2a;
```
split this background rule into pieces. Ex `background-image:` etc.